json_include.py

```diff
@@ -34,7 +40,7 @@ def walk_through_to_include(o, dirpath):
     if isinstance(o, dict):
         is_include_exp = False
         if set(o) == set([INCLUDE_KEY]):
-            include_name = get_include_name(o.values()[0])
+            include_name = get_include_name(list(o.values())[0])
```

Comment:
Since `.values()` in 2 doesn't behave the same in 3, to make dict key/value iteration 2/3 compatible, we need an extra function and also a 2, 3 flag to handle, my suggestion:

```
PY2 = sys.version_info[0] == 2

def dict_view(method):
    n = method.__name__
    d = method.__self
    if PY2:
        return getattr(d, 'view' + n)()
    else:
        return method()
```

then we can use `for i in dict_view(o.values)` in both 2 and 3 to iter over the value,
and `list(dict_view(o.values))` to get a list of values directly.

so 蛋疼！！！